testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/cli/ReloadOpsTestCase.java

```diff
@@ -91,7 +77,7 @@ public void testWriteAttribvuteWithReload() throws Exception {
         result = cli.readAllAsOpResult();
         assertTrue(result.isIsOutcomeSuccess());
         checkResponseHeadersForProcessState(result);
-        reloadServer(TIMEOUT);
+        container.reload(TIMEOUT);
```

Comment:
Just remembered why I did this, the original code for cli reload does
final CommandContext ctx = CLITestUtil.getCommandContext();
which creates new network connection to http-remoting port. and than executes reload.

The replacement code uses already open client and reuses that. 
So in short it speeds up the test.